packages/lab/stories/typography/BaseText.stories.tsx

```diff
@@ -4,7 +4,7 @@ import { Text, TextProps } from "@jpmorganchase/uitk-lab";
 import { useEffect, useState } from "react";
```

Comment:
Should the stories folder also change to /text?

Reply:
also does this need a changeset?

Reply:
Also the e2e tests folder

Reply:
Whoops, I made the change locally but it wasn't picked up...

I don't think it needs a changeset as it's not changing any user facing code. Only refactoring the structure and we don't add changelogs for docs changes at the moment.